PIL/Image.py

```diff
@@ -538,12 +573,15 @@ def _new(self, im):
 
     # Context Manager Support
     def __enter__(self):
+        # type: () -> Image
         return self
 
     def __exit__(self, *args):
+        # type: (*Any) -> None
```

Comment:
I'm not too familiar with __exit__, but I believe in python2 that `*args` can only be one type?
If not, that's certainly plausible, particularly if changing the function signature (which I've been avoiding).